Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -162,50 +161,73 @@ void ABenchmarkGymGameModeBase::TickPlayersConnectedCheck(float DeltaSeconds)
 		return;
 	}
 
-	if (SecondsTillPlayerCheck > 0.0f)
+	// Only check players once
+	if (bHasDonePlayerCheck)
```

Comment:
We also want to catch players disconnecting during the game, so I don't think we should have this flag.

Reply:
Oh - we didn't do that previously - that seems incorrect.

Reply:
The native fail log should be in GameMode::Logout too